### PR TITLE
Analysis template implementation complete

### PR DESCRIPTION
**Pull request by jpbulloch5**
The gateway now successfully implements the analysis.yml template from the project-registry-devops repo.
Depends on: https://github.com/1053-August-Duet-Project-Registry/project-registry-devops/pull/3

### CODE REVIEW

**Review by BeoHawk25 — DISMISSED** [review]

**Review by escalonn — CHANGES_REQUESTED** [review]
cf. https://github.com/1053-August-Duet-Project-Registry/project-registry-account-microservice/pull/3#pullrequestreview-711097506

**Review by escalonn — CHANGES_REQUESTED** [review]
see https://github.com/1053-August-Duet-Project-Registry/project-registry-account-microservice/pull/3#pullrequestreview-711258014
also, the build is failing (and it should be a required check via branch policy).

File: devops/azure-pipelines.yml [hunk 1]
```diff
@@ -9,9 +10,10 @@ resources:
     ref: refs/heads/feature-templates
 
 # This pipeline will trigger on commits or pull-requests made to the branch-dev
-# or main branches.  Note that static analysis with SonarCloud is only performed
-# on "long-lived" branches, which are designated as 'main' and any branch name
-# prefixed with 'branch-'.
+# or main branches.  Note that full static analysis with SonarCloud is only 
+# performed on "long-lived" branches, which are designated as 'main' and any 
+# branch name prefixed with 'branch-'.  Minimal analysis is still performed on
+# "short-lived" branches and special analyisis is performed on pull-requests
```
Comment (escalonn) [inline devops/azure-pipelines.yml]: spelling

**Review by escalonn — DISMISSED** [review]

**Comment by escalonn** [comment]
see previous review. @BeoHawk25 the pipeline is still not a required check on these PRs in these repos. the sonar gate should be too

File: devops/azure-pipelines.yml (hunk 1 above)
Comment (jpbulloch5) [reply devops/azure-pipelines.yml]: updated grammar/spelling on comment block

**Review by escalonn — APPROVED** [review]

**Review by BeoHawk25 — APPROVED** [review]

**Pull request merged**